docs/processes/release/tasks/versions.md

```diff
@@ -28,6 +28,26 @@ For reference, see the section in the User Guide about Version numbers here:
 
 <http://userguide.icu-project.org/design#TOC-Version-Numbers-in-ICU>
 
+## ICU Version Number for Front load, RC and GA tasks
+
+The process of releasing a new ICU version (E.G. ICU 70.1) is divided in three phases:
+* Front loading tasks
+* Release Candidate (RC) tasks
+* General Availability (GA) tasks
+
+As of ICU 70, the ICU Version number changes for each of these tasks.
+
+When "front loading" tasks, the version number will consist of a Major number, Minor number, and a Patch number. 
+For example: ICU version 70.0.1
+
+For the RC and GA tasks, the ICU version number will consist of a Major number and a Minor number.
+For example: ICU version 70.1
+
+This means that when updating from the front load tasks to the RC tasks, files such as 
+[icu4c/source/common/unicode/uvernum.h](https://github.com/unicode-org/icu/blob/main/icu4c/source/common/unicode/uvernum.h)
+need to be correspondingly updated. See below for more files to be updated and steps to be followed.
```

Comment:
Thanks for adding this documentation! :)

Reply:
Thanks Erik!